build/manifest.json

```diff
@@ -1,19 +1,19 @@
 {
   "short_name": "Gurkult",
-  "name": "An Open Source Community",
+  "name": "Gurkult - A Kult for open source",
```

Comment:
The wording here was very intentional. We are straying away from advertising as a `cult` as it is misleading and not really the message we wish to convey.